dAuthServer/AuthServer.cpp

```diff
@@ -83,12 +83,15 @@ int main(int argc, char** argv) {
 	Game::randomEngine = std::mt19937(time(0));
 
 	//It's safe to pass 'localhost' here, as the IP is only used as the external IP.
-	uint32_t maxClients = 50;
+	uint32_t maxClients = 999;
 	uint32_t ourPort = 1001; //LU client is hardcoded to use this for auth port, so I'm making it the default.
-	if (Game::config->GetValue("max_clients") != "") maxClients = std::stoi(Game::config->GetValue("max_clients"));
-	if (Game::config->GetValue("auth_server_port") != "") ourPort = std::atoi(Game::config->GetValue("auth_server_port").c_str());
+	std::string ourIP = "localhost";
 
-	Game::server = new dServer(Game::config->GetValue("external_ip"), ourPort, 0, maxClients, false, true, Game::logger, masterIP, masterPort, ServerType::Auth, Game::config, &Game::lastSignal);
+	if (!Game::config->GetValue("max_clients").empty()) maxClients = std::stoi(Game::config->GetValue("max_clients"));
```

Comment:
I know it's a oneliner this way, but can we only do the GetValue once per key?

Reply:
Was thinking about doing that for all of these I've touched. Since they're gotten twice at worst case. 
Want me to change that and cut it down to once?

Reply:
I think I'm just gong to leave it as it unless others bring it up. I'm not sure of the impact of storing it locally vs just grabbing it every time

Reply:
Storing it locally as a copy is safe to do.

Reply:
(would preferably use tryparse as we discussed, so I'll try to look into why that is causing issues more later)

Reply:
move things to use tryParse where applicable